packages/models/src/_base/index.ts

```diff
@@ -1,4 +1,4 @@
 export type BaseEntity = {
   id: string;
-  runId: string;
+  runId?: string;
```

Comment:
let's instead fix the root cause by removing all `.default("initial-run-id")` assurances in the code, then putting `runId` back to being required
```suggestion
  runId: string;
```

Reply:
this will cause a tsc error when upserting in the search service, because the inferred `*Row` has an optional `runId` which does not satisfy the `BaseEntity`